corpus/pages/views.py

```diff
@@ -41,3 +45,38 @@ def sig(request, sig_name):
     }
 
     return render(request, "pages/sig.html", args)
+
+
+def team(request):
+    compsoc_members = ExecutiveMember.objects.filter(
+        (Q(core__isnull=True) | Q(core=None))
```

Comment:
Why are both `core__isnull=True` and `core=None` being checked? Is one not sufficient?

Reply:
I looked it up and apparently they're not the same in postgres JSONFields. So i'm not exactly sure if we should be using both of them or not

[https://stackoverflow.com/questions/29858701/django-query-isnull-true-or-none](url)